apps/extension/FIREFOX_README.md

```diff
@@ -63,25 +44,16 @@ This build was produced using the following environment:
 - Ubuntu 24.04 LTS (Desktop edition)
```

Comment:
After this is tested & merged, I can take out the Ubuntu dependency & specs below, they shouldn't matter. With Docker, we shouldn't have any compatibility issues, hopefully, though I'd like to test it on ARM & AMD64!